sorts/merge_insertion_sort.py

```diff
@@ -0,0 +1,106 @@
+"""
+This is a pure Python implementation of the merge-insertion sort algorithm
+Source: https://en.wikipedia.org/wiki/Merge-insertion_sort
+
+For doctests run following command:
+python -m doctest -v merge_insertion_sort.py
+or
+python3 -m doctest -v merge_insertion_sort.py
```

Comment:
```suggestion
python3 -m doctest -v merge_insertion_sort.py
or
python -m doctest -v merge_insertion_sort.py
```
Let's do python3 first because python might still point to legacy Python.